packages/devtools_app/lib/src/service/isolate_manager.dart

```diff
@@ -113,15 +112,15 @@ class IsolateManager extends Disposer {
 
   Future<void> _loadIsolateState(IsolateRef isolateRef) async {
     final service = _service;
-    var isolate = await _service.getIsolate(isolateRef.id);
-    if (!isolate.runnable) {
+    var isolate = await _service!.getIsolate(isolateRef.id!);
+    if (!isolate.runnable!) {
```

Comment:
let's do `if (isolate.runnable == false)` so that the null case is covered

Reply:
done